automation_tools/__init__.py

```diff
@@ -1726,6 +1729,8 @@ def satellite6_upgrade(admin_password=None):
     # Stop katello services, except mongod
     run('katello-service stop')
     run('service-wait mongod start')
+    if major_ver == 7:
+        run('service tomcat stop')
```

Comment:
Use manage_daemon helper. It will ensure to run the right service management utility depending on the rhel version.